tests/test.linter.js

```diff
@@ -899,21 +899,24 @@ describe('Linter.extractMetadata()', function() {
     var addonLinter = new Linter({
       _: ['tests/fixtures/angular-unadvised-library.xpi'],
```

Comment:
You should probably change this line to 'fake.xpi' or something since you're not actually looking into the fixture xpi ata all any more. Also all the comments above can be removed.